modes/dired/evil-collection-dired.el

```diff
@@ -186,7 +186,17 @@
     ";d" 'epa-dired-do-decrypt
     ";v" 'epa-dired-do-verify
     ";s" 'epa-dired-do-sign
-    ";e" 'epa-dired-do-encrypt))
+    ";e" 'epa-dired-do-encrypt)
+
+    ;; dired-subtree commands
+    (with-eval-after-load 'dired-subtree
+	(evil-collection-define-key 'normal 'dired-mode-map
+	    "I" 'dired-subtree-toggle
```

Comment:
"I" may be too intrusive since it's replacing:

```
(dired-maybe-insert-subdir DIRNAME &optional SWITCHES NO-ERROR-IF-NOT-DIR-P)
```

which I think can be useful even with dired-subtree.

Reply:
If the replacement is the issue, we could bind `M-i` to `dired-subtree-toggle`. The key `i` for `insert` is just too tempting not to use.

Reply:
Yeah, I think I is too much, don't really have an opinion on toggle. I actually think most people probably just use only one of these functions and wouldn't bother with the other.

Maybe a defcustom on the one to bind works too.

Reply:
A defcustom for a single key barely makes sense, I think. Maybe if it bundles some other functions, something like "prefer subtree over subdirectory" or such.

Reply:
Sure, I'm not sure about M-i though, do you use dired-subtree-toggle?

I think we can just leave it out, probably.

Reply:
Exclusively. I think I'll just leave it out for now, we can revisit the keybindings in the future.

My main goal was to get one more package into the collection for review and consistency.

I'll fix my typo later and amend the commits to reflect the feedback a bit later.